routesrv/testdata/redis-ip.json

```diff
@@ -0,0 +1 @@
+{"Endpoints":[{"Url":"10.2.5.7:6379"},{"Url":"10.2.55.7:6379"}]}
```

Comment:
No newline to match json response

Reply:
Remove newline with https://github.com/zalando/skipper/blob/ec12fa30dcf422a9bcbfce42e033d6ba7a7befa7/routesrv/routesrv_test.go#L95-L103